tests/functional/fixtures/docker-compose.yml

```diff
@@ -45,7 +45,7 @@ services:
       - gitlab-network
 
   gitlab-runner:
-    image: gitlab/gitlab-runner:latest
+    image: '${GITLAB_RUNNER_IMAGE}:${GITLAB_RUNNER_TAG}'
```

Comment:
Wow that's a lot of config hehe, thanks. I guess we won't need this often but doesn't hurt :slightly_smiling_face:

Reply:
Oh, I was copying how there is a `GITLAB_IMAGE` and `GITLAB_TAG` in the same file. Do you want to change it to just be a hard-coded version?

Reply:
All good, I just didn't expect we'd need to customize this, but we can keep it the same way as here just in case!